src/react-apps/applications/runtime/src/features/instantiate/containers/index.tsx

```diff
@@ -170,7 +169,12 @@ function InstantiateContainer(props: IServiceInfoProps) {
   } else {
     return (
       <>
-        <AltinnAppHeader profile={profile} language={language}/>
+        <AltinnAppHeader
+          logoColor={AltinnAppTheme.altinnPalette.primary.blueDarker}
+          headerBackgroundColor={AltinnAppTheme.altinnPalette.primary.blue}
+          party={selectedParty}
```

Comment:
No nullcheck for selectedParty? It can be null on first load.

Reply:
Not needed, it can be passed as null  👍